Sources/XLPagerTabStrip/ButtonBarPagerTabStripViewController.swift

```diff
@@ -106,37 +106,30 @@ open class ButtonBarPagerTabStripViewController: PagerTabStripViewController, Pa
     open override func viewDidLoad() {
         super.viewDidLoad()
         
-        var bundle = Bundle(for: ButtonBarViewCell.self)
-        if let resourcePath = bundle.path(forResource: "XLPagerTabStrip", ofType: "bundle") {
-            if let resourcesBundle = Bundle(path: resourcePath) {
-                bundle = resourcesBundle
-            }
-        }
-        
-        buttonBarItemSpec = .nibFile(nibName: "ButtonCell", bundle: bundle, width: { [weak self] (childItemInfo) -> CGFloat in
-                let label = UILabel()
-                label.translatesAutoresizingMaskIntoConstraints = false
-                label.font = self?.settings.style.buttonBarItemFont
-                label.text = childItemInfo.title
-                let labelSize = label.intrinsicContentSize
-                return labelSize.width + (self?.settings.style.buttonBarItemLeftRightMargin ?? 8) * 2
+        buttonBarItemSpec = .nibFile(nibName: "ButtonCell", bundle: .module, width: { [weak self] (childItemInfo) -> CGFloat in
```

Comment:
Bundle.module only give the good path when the package is imported via SPM, isn't it ? 
I am afraid that it breaks the access to the bundle with Cocoapods

Reply:
Because the name of the resource bundle is ** XLPagerTabStrip** in your pod spec, but I think that will be **XLPagerTabStrip_ XLPagerTabStrip** in Bundle.module

Reply:
I kept the old implementation and only use Bundle.module for the swift package now